prime-router/docs/design/design/rs-ti-integration.md

```diff
@@ -0,0 +1,13 @@
+# Report Stream and Intermediary Integration
+
+## ETOR
+
+For the purpose of supporting our partners on the ETOR initiative, there is a need to make certain data available to our
```

Comment:
Can we be more detailed? What is the "certain data" and why was a new endpoint needed? Why do the existing delivery and history endpoints not cut it?

Reply:
Fixed.